lib/me-keyword.resource

```diff
@@ -0,0 +1,22 @@
+*** Settings ***
+Library     BuiltIn
+Library     Process
+
+
+*** Keywords ***
+Check ME State
+    [Documentation]    Keyword Checks The ME PCI Register Offset 0x40 To Find Current ME State, Returns:
+    ...    -Disabled(soft) For 3
+    ...    -Disabled(HAP) for 2
+    ...    -Enabled for 0
+
+    ${out}=    Execute Command In Terminal    setpci -s 16.0 40.L
+    ${char}=    Evaluate    ${out}[3]
```

Comment:
We're converting a 32bit integer to a string, and then getting the 3rd character from the left, a rather roundabout way to read bits 16:19 of a register :). Let's add a comment explaining what's going on here, something like:

```
# Current Operation Mode: bits 16:19 of register HFSTS1 (0x40)
```

Reply:
will the comment be sufficient then? if so here's the update: https://github.com/Dasharo/open-source-firmware-validation/pull/375/commits/e8738cef4b6e89953cb45972e3df68a60965bd21